apps/web/src/App.tsx

```diff
@@ -1,255 +1,10 @@
-                  <Route path={ROUTES.QUICK_START_SETUP_SUCCESS} element={<InAppSuccess />} />
-                  <Route path={ROUTES.ACTIVITIES} element={<ActivitiesPage />} />
-                  <Route path={ROUTES.SETTINGS} element={<SettingsPage />}>
-                    <Route path="" element={<ApiKeysCard />} />
-                    <Route path="email" element={<EmailSettings />} />
-                    <Route
-                      path="permissions"
-                      element={
-                        <ProductLead
-                          icon={<UserAccess />}
-                          id="rbac-permissions"
-                          title="Role-based access control"
-                          text="Securely manage users' permissions to access system resources."
-                          closeable={false}
-                        />
-                      }
-                    />
-                    <Route
-                      path="sso"
-                      element={
-                        <ProductLead
-                          icon={<SSO />}
-                          id="sso-settings"
-                          title="Single Sign-On (SSO)"
-                          text="Simplify user authentication and enhance security."
-                          closeable={false}
-                        />
-                      }
-                    />
-                    <Route
-                      path="data-integrations"
-                      element={
-                        <ProductLead
-                          icon={<Cloud />}
-                          id="data-integrations-settings"
-                          title="Data Integrations"
-                          text="Share data with 3rd party services via Segment and Datadog integrations to monitor analytics."
-                          closeable={false}
-                        />
-                      }
-                    />
-                  </Route>
-                  <Route path={ROUTES.INTEGRATIONS} element={<IntegrationsListPage />}>
-                    <Route path="create" element={<SelectProviderPage />} />
-                    <Route path="create/:channel/:providerId" element={<CreateProviderPage />} />
-                    <Route path=":integrationId" element={<UpdateProviderPage />} />
-                  </Route>
-                  <Route path={ROUTES.TEAM} element={<MembersInvitePage />} />
-                  <Route path={ROUTES.CHANGES} element={<PromoteChangesPage />} />
-                  <Route path={ROUTES.SUBSCRIBERS} element={<SubscribersList />} />
-                  <Route path={ROUTES.BRAND} element={<BrandPage />}>
-                    <Route path="" element={<BrandingForm />} />
-                    <Route path="layouts" element={<LayoutsListPage />} />
-                  </Route>
-                  <Route path="/translations/*" element={<TranslationRoutes />} />
-                </Route>
-              </Routes>
```

Comment:
Moved to `AppRoutes.tsx`

Separating theses from the Providers enables us to use feature flags at the route-level, which is great for large page change features like this

Reply:
finally! thank you so much Joel! 🔥